setup/sql/ilDBTemplate.php

```diff
@@ -33947,7 +33947,6 @@ function setupILIASDatabase()
 	)
 	,"insert_time" => array (
 		"notnull" => true
-		,"default" => "NOW()"
```

Comment:
@chfsx: Isn't this due to any changes in the reflection implementation? This issue did not occur in 5.1. I did not make any changes to this table. So imo it is caused by either a) changes in the reflection implementation or b) in the way anton creates the db templates. If anton creates these templates using empty existing installations (which just run DB update), changes that we make in them will be overwritten next time.